deployment/misc/sdklocalmethodrunner_dev.sh

```diff
@@ -13,7 +13,7 @@ if [ -e /etc/clustername ] ; then
     KBASE_ENDPOINT=$2
     BASE_DIR=$KBASE_RUN_DIR/jobs/$JOBID
     mkdir -p $BASE_DIR && cd $BASE_DIR
-    java -cp $NJSW_JAR us.kbase.narrativejobservice.sdkjobs.SDKLocalMethodRunner $JOBID $KBASE_ENDPOINT > sdk_lmr.out 2> sdk_lmr.err
+    exec java -cp $NJSW_JAR us.kbase.narrativejobservice.sdkjobs.SDKLocalMethodRunner $JOBID $KBASE_ENDPOINT > sdk_lmr.out 2> sdk_lmr.err
```

Comment:
I copied the exact contents of sdkmethodrunner.sh into sdkmethodrunner_dev.sh  and made sure mini_kb=true